src/passes/Print.cpp

```diff
@@ -78,6 +78,16 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     return name;
   }
 
+  std::ostream& printName(Name name) {
+    // we need to quote names if they have tricky chars
```

Comment:
This doesn't match the spec parser; using parens in the internal name is a syntax error.